src/Feature/SCM/Feature_SCM.re

```diff
@@ -398,24 +398,32 @@ open Revery.UI.Components;
 
 module Input = Oni_Components.Input;
 
+module Theme = Feature_Theme;
+
+module Colors = {
+  let foreground = Theme.Colors.SideBar.foreground;
+  let background = Theme.Colors.SideBar.background;
+  let hoverBackground = Theme.Colors.List.hoverBackground;
+};
```

Comment:
Aliasing colors locally makes them more convenient to use, gives local contextual meaning, and also gives a nice overview of which colors are used in this module.

Reply:
I like it 👍